### DISCUSSION TITLE
Warning: The plugin "removeAttrs" requires the "attrs" parameter.

### DISCUSSION THREAD

**Discussion by keegankb93** [gatsbyjs/gatsby#33513] [Help] [answered]
I was wondering if anyone has come across this in V4?

```
Warning: The plugin "removeAttrs" requires the "attrs" parameter.
It should have a pattern to remove, otherwise the plugin is a noop.
Config example:

plugins: [
  {
    name: "removeAttrs",
    params: {
      attrs: "(fill|stroke)"
    }
  }
]
```

Only happens on _first build_ after clearing the cache. It looks like it's related to the svgo node_module. I'm not able to find much documentation on fixes other than a yml file. I can attempt to edit it accordingly, but I'm not sure how Gatsby is incorporating this module into its framework. It looks like I can more than likely ignore it, but I'd prefer to handle it if possible.

**Comment by LekoArts** [comment, marked as answer]
This was fixed with https://github.com/gatsbyjs/gatsby/pull/33266 - make sure you're on the latest version

**Reply by keegankb93** [reply]
Ahhh, thank you!!
